pep-0458.txt

```diff
@@ -369,6 +369,12 @@ generate cryptographic keys, and manage a TUF repository.
 
 __ https://github.com/theupdateframework/tuf/blob/v0.11.1/docs/TUTORIAL.md#how-to-create-and-modify-a-tuf-repository
 
+In standard operation the *bin-n* metadata will be created and signed as new
+distributions are uploaded to PyPI. However, there will also need to be a
```

Comment:
Do we "packages" or "distributions"? We have a mix of the two now. Let's be consistent throughout, please. Maybe we need another PR. @jhdalek55

Reply:
Seems like "packages" is much more frequently used. I went with "distributions" here as it's a term we define in the _Definitions_ section. We should definitely be consistent throughout.

Reply:
I did a simple search and found 20 occurrences of "distribution" and 37 occurrences of "package"

Reply:
Thanks for checking, Joshua!

Re: both 🤦‍♂ 

We'll need to fix it separately...